web/src/features/comments/CommentCountIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { MessageCircleMore } from "lucide-react";
+
+export function CommentCountIcon({ count }: { count?: number }) {
+  if (!count) return null;
+
+  return (
+    <span className="relative mr-1 text-xs">
+      <MessageCircleMore className="h-5 w-5 text-muted-foreground" />
+      <span className="absolute -top-0.5 left-3 flex h-4 w-4 items-center justify-center rounded-full border border-muted-foreground bg-accent-light-blue text-xs">
+        {count}
```

Comment:
does not look good if count is too large for fixed-pixel circle. I'd rather use smaller font, fixed padding, and rounded-full -> becomes more like a pill if there are e.g. 120 comments